decipher.go

```diff
@@ -37,13 +37,13 @@ func (y *Youtube) parseDecipherOpsAndArgs() (operations []string, args []int, er
 	}
 	embeddedPage := string(embeddedPageBodyBytes)
 
-	playerConfigPattern := regexp.MustCompile("yt\\.setConfig\\({'PLAYER_CONFIG':(.*)}\\);")
+	playerConfigPattern := regexp.MustCompile(`yt.setConfig({'PLAYER_CONFIG':(.*)});`)
```

Comment:
test failed.  I guess here should be  `yt\.setConfig\({'PLAYER_CONFIG':(.*)}\);`
still need '\' for regex.

Reply:
thanks a lot. you are right!!!